docs/README.rst

```diff
@@ -1,5 +1,5 @@
-F5 Marathon BIG-IP Controller
-=============================
+|project|
```

Comment:
We shouldn't use substitutions for document titles because the substitution shows up in google searches.